components/GymPlan.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+
+export default function GymPlan() {
+  return (
+    <>
+      <StyledHeadline> Gym Plan </StyledHeadline>
+      <StyledSection>
+        <StyledHeadlineSection> Warm-Up </StyledHeadlineSection>
+        <form>
+          <StyledInputNumber type="number" name="set" placeholder="2" />
```

Comment:
your input fields have no labels. eg. <label htmlFor="number"></label>.
I think this is also an semantic and ally thing you may consider. Or use at least aria-labels.